civihr_default_theme/includes/header.inc

```diff
@@ -0,0 +1,100 @@
+<?php
+
+$civicrm_access = user_access("access CiviCRM");
+$admin_link = l(t('CiviHR admin'), 'civicrm/tasksassignments/dashboard', ['fragment' => '/tasks']);
+$options = array('html' => TRUE);
+$ssp_link = l(t('CiviHR SSP'), 'dashboard', $options);
+
+$resourceTypeVocabularyID = taxonomy_vocabulary_machine_name_load('hr_resource_type')->vid;
+$mapGearLinks = [
+  [
+    'permissions' => ["access content overview"],
+    'link' => l(t('Manage documents'), 'admin/content', $options),
+  ],
+  [
+    'permissions' => ["administer users", "access users overview"],
```

Comment:
please use single quotes instead of double here and other places in the `mapGearLinks` wherever if its not being wrapped around a variable.